src/main/java/com/ifelseco/userapi/controller/RegisterController.java

```diff
@@ -66,4 +72,53 @@ public ResponseEntity register(@RequestBody RegisterModel registerModel) throws
 
     }
 
+    @PutMapping
+    public ResponseEntity<BaseResponseModel> updateUserInfo(@RequestBody UpdateModel updateModel
+    ) {
+
+        //updateModel :)
+
+        BaseResponseModel baseResponseModel= new BaseResponseModel();
+
+
+
+        try {
+
+            User currentUser = userService.findByEmail(updateModel.getEmail());
+
+
+
+            String firstName = updateModel.getFirstName();
+            String email = updateModel.getEmail();
+            String lastName = updateModel.getLastName();
+            String userName = updateModel.getUserName();
+
+
+
+            // is email exist?
+            if (currentUser == null) {
+                return new ResponseEntity("User is not found", HttpStatus.BAD_REQUEST);
```

Comment:
you return BaseResponseModel so set baseResponseModel and return it via ResponseEntity